src/malco/post_process/compute_mrr.py

```diff
@@ -42,17 +59,34 @@ def compute_mrr(output_dir, prompt_dir, correct_answer_file) -> Path:
         # df['correct_term'] is an OMIM
         # call OAK and get OMIM IDs for df['term'] and see if df['correct_term'] is one of them
         # in the case of phenotypic series, if Mondo corresponds to grouping term, accept it
-        df['is_correct'] = df.apply(
-            lambda row: score_grounded_result(row['term'], row['correct_term']) > 0,
-            axis=1)
+                        
+        #df['is_correct'] = df.apply(
+        #       lambda row: score_grounded_result(row['term'], row['correct_term'], mondo) > 0,
+         #   axis=1)
 
```

Comment:
it is good form not to leave commented out code unless there is a really good reason!

Reply:
Sure, thanks! I am not 100% sure this is really necessary